services/director-v2/tests/unit/with_dbs/test_modules_dynamic_sidecar_docker_service_specs.py

```diff
@@ -116,7 +116,7 @@ def expected_dynamic_sidecar_spec(
                         "is_service_environment_ready": False,
                         "service_removal_state": {
                             "can_remove": False,
-                            "can_save": None,
```

Comment:
why was this alraedy here? Is ``can_save`` still nullable?

Reply:
Because this was set when closing. But now it is set when creating, it can be just bool and no longer None|bool.